components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -1218,6 +1266,14 @@ void ConversationDriver::RateMessage(
     bool is_liked,
     uint32_t turn_id,
     mojom::PageHandler::RateMessageCallback callback) {
+  auto& model = GetCurrentModel();
+
+  // Disable rating for custom models
+  if (model.option->is_custom_model()) {
+    std::move(callback).Run(std::nullopt);
+    return;
+  }
```

Comment:
Should we just `CHECK` here since the UI shouldn't allow rating messages that aren't Leo models?

Reply:
yes, this makes sense